### ISSUE TITLE
Automatic refresh on Dolphin browser

### ISSUE THREAD

**Issue by PugaBear** [thelounge/lounge#336]
I had to replace Chrome with the Dolphin browser on my Android (4.2.2) tablet due to space issues. I then found that both my public and private lounge pages refresh automatically every 5 seconds. Obviously, this makes it impossible to do anything. 

I dont know if this is fixable on your end but worth looking into.

I tried to get someone to reproduce this before submitting an issue, but everyone was busy/had an iPhone.

**Comment by maxpoulin64** [comment]
I just tried it, and was able to reproduce as well as fix it.

I disabled two settings in it: smart cache, and page preloading (which defaults to over WiFi only). Then after restarting the browser it worked fine, so either of those two settings are probably the culpit.

I will therefore mark this bug as invalid, as it's clearly a browser bug. Sorry, but I doubt there's anything we can do as it's socket.io that handles all the lingpolling/websocket stuff. I would recommend trying with the stock browser instead.
